app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -250,6 +193,97 @@
       z-index: 11;
     }
   }
+
+  &__bar {
+    @include border-radius-small;
+    width: 10rem;
+    position: absolute;
+    right: 1rem;
+    z-index: 100;
+    background-color: $vocab-bg;
+    overflow: hidden;
+    height: 90vh;
+    display: none;
+
+    &:after {
+      content: '';
+      position: absolute;
+      z-index: 2;
+      bottom: 0;
+      left: 0;
+      width: 100%;
+      height: 3rem;
+      background: linear-gradient(transparent, $vocab-bg);
+    }
+
+    &-header {
+      background-color: $vocab-header-bg;
+      padding: .5rem 0 1rem;
+    }
+
+    &-list {
+      padding: .5rem .3rem 1rem;
+      margin: 0;
+      list-style-type: none;
+      height: 78vh;
+      overflow: hidden;
+      overflow-y: auto;
+      box-shadow: inset 0 1px .125rem $light-med-grey;
+    }
+
+    &-item {
+      @include border-radius-small;
+    
+      background-color: $white;
+      width: 100%;
+      margin: 1rem 0;
+      box-shadow: 0 .125rem .25rem $light-med-grey;
+      box-sizing: border-box;
+    
+      .remove {
+        top: -8px;
+        right: -10px;
+      }
+    }
+
+    &-image {
+      min-height: 5rem;
+    }
+  
+    .flash {
+      @include border-radius-small;
+    
+      margin: 0 auto;
+      background-color: $success-green;
+      width: 80%;
+      padding: .25rem;
+      opacity: 0;
+      max-height: 0;
+      transition: {
```

Comment:
I've seen a few transitions around in the last couple of PRs is there much consistency in their settings? If so it might be time to start looking at making a little mixin for the properties.

Reply:
Yeah, they're all the same. Mixin time!